pytest_dockerctl/__init__.py

```diff
@@ -13,6 +13,16 @@ class DockerClientError(Exception):
     """Generic docker API error.
     """
 
+def has_attr(cntr, attr_path) -> bool:
```

Comment:
not that there's any typing requirements in this repo yet (😂 ) but maybe let's start here? For one, knowing what `attr_path` is would be handy to grok what this is doing.